app/controllers/comments_controller.rb

```diff
@@ -28,6 +28,13 @@ def update
     end
   end
 
+  def destroy
+    comment = Comment.find_by!(giveaway_id: params[:giveaway_id], id: params[:id])
```

Comment:
To DRY the code, we can include destroy action in `before_action :find_comment, only: %i[edit update]`